assets/js/index.js

```diff
@@ -0,0 +1,40 @@
+const addTaskIcon = document.getElementById("add-icon");
+const addTaskInput = document.getElementById("input");
+const errorMessage = document.getElementById("error-message")
+const backendErrorMessage = document.getElementById("backend-error-message");
+
+async function postTodo(todoText) {
+  const url = "http://localhost:3030/todos";
+  
+  try {
+    const response = await fetch(url,{
+      method: "POST",
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({text: todoText})
+    });
+    if (response.status === 400) {
```

Comment:
What if you get status of `500`? Please check for the correct status and show error in case of any other status.

Reply:
Done.